### PR TITLE
Bump webhook to v0.4.0-rc2

### PR DESCRIPTION
**Pull request by cmurphy**

### CODE REVIEW

**Comment by diogoasouza** [comment]
You need to add the new chart version to the `releases.yaml` file

**Review by diogoasouza — DISMISSED** [review]

File: packages/rancher-webhook/package.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-url: https://github.com/rancher/webhook/releases/download/v0.3.5/rancher-webhook-0.3.5.tgz
+url: https://github.com/rancher/webhook/releases/download/v0.4.0-rc1/rancher-webhook-0.4.0-rc1.tgz
 version: 103.0.0
```
Comment (maxsokolovsky) [inline packages/rancher-webhook/package.yaml:2]: Aren't we also bumping the `version` to 104.0.0?

Comment (maxsokolovsky) [reply packages/rancher-webhook/package.yaml:2]: Never mind, just saw a thread offline.

**Comment by cmurphy** [comment]
Updated to use webhook rc2 which aligns the Chart.yaml rancher version

**Review by diogoasouza — APPROVED** [review]

**Review by cbron — APPROVED** [review]

**Pull request merged**